src/design/plone/contenttypes/behaviors/evento.py

```diff
@@ -123,7 +88,7 @@ class IEvento(model.Schema):
     # ref
     luogo_event = RelationList(
```

Comment:
Uniformiamo i nomi dei campi: se li facciamo tutti italianeggianti, allora questo lo cambierei in "luogo_evento" per esempio